sdk/python/feast/templates/aws/bootstrap.py

```diff
@@ -1,3 +1,8 @@
+import inquirer
```

Comment:
Out of curiousity, is there anything `inquirer` gives us that `Click` doesn't?

Reply:
+1 to fewer deps if possible.